.travis.yml

```diff
@@ -1,20 +1,52 @@
 language: python
+
 python:
 - '2.7'
+
+install:
+- pip install pipenv
```

Comment:
Oh I see now what you meant - we could remove this and `pipenv run` below to make things faster, when that's the only use case.